roles/scaffold_tests/files/module_name/tasks/main.yaml

```diff
@@ -0,0 +1,11 @@
+- name: Collect common test cases
+  find:
+    paths: "{{ role_path }}/tests/common"
```

Comment:
@cidrblock Just a thought, since we're already having a sub-directory under `tests/` (`common`) can we name it to `cli` so that down the line when (or if) we have other implementations we wouldn't need to refactor the tests. Plus, this would kind of keep us at sync with what we have in ansible/ansible now.

Reply:
Oh, my bad. So we're gonna have common tests under `common` sub-directory and then cli specific tests under `cli` sub-directory. The {{ operation }} files under `scaffold_tests/files/module_name/tests/common` should be `scaffold_tests/files/module_name/tests/cli` then?

Reply:
`common/` directory was meant to be for the tests that will run for multiple transports. So instead of having same tests in <api>/ and cli/ we decided to have common/ dir. But I guess we do not need common/ dir as of now as we are going to support only `network_cli` in the initial phase.

Reply:
@cidrblock @NilashishC ^